src/accelerate/accelerator.py

```diff
@@ -1443,9 +1443,16 @@ def prepare_model(self, model: torch.nn.Module, device_placement: bool = None, e
                             "you're training on. Make sure you loaded the model on the correct device using for example `device_map={'':torch.cuda.current_device()}` or `device_map={'':torch.xpu.current_device()}`"
                         )
 
-            if "cpu" in model_devices or "disk" in model_devices:
+            bnb_multi_backends = False
+            try:
+                from transformers.utils import is_bitsandbytes_multi_backend_available
+                bnb_multi_backends = is_bitsandbytes_multi_backend_available()
+            except ImportError:
+                bnb_multi_backends = False
```

Comment:
Let's not use the transformers import in accelerate. You can just create the same function in accelerate. This way, we don't need to have the try/catch also.

Reply:
Right. Done.